lambdas/profiles/read/index.js

```diff
@@ -5,6 +5,7 @@ const s3 = new S3({
 
 module.exports.handler = async function handler ({
   event,
+  context,
   headers,
   pathParameters
 }) {
```

Comment:
still wrong handler signature;; you are destructuring the first argument to the handler
should be: `handler(event, context)`
`event` structure visible [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/37ef96abd4c38621aebe374ed67a3f38a51f55f3/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L65)